+readme Cross link to extras (#88)
apple/swift-metrics: README.md

- [StatsD Client](https://github.com/apple/swift-statsd-client), support for StatsD
- Your library? [Get in touch!](https://forums.swift.org/c/server)

### Swift Metrics Extras

> You may also be interestedin some "extras" modules which are collected in the [Swift Metrics Extras](https://github.com/apple/swift-metrics-extras) repository.

Author: ```suggestion
You may also be interested in some "extra" modules which are collected in the [Swift Metrics Extras](https://github.com/apple/swift-metrics-extras) repository.
```

- [StatsD Client](https://github.com/apple/swift-statsd-client), support for StatsD
- Your library? [Get in touch!](https://forums.swift.org/c/server)

### Swift Metrics Extras

> You may also be interestedin some "extras" modules which are collected in the [swift metrics extras](https://github.com/apple/swift-metrics-extras) repository.

Reviewer: ```suggestion
You may also be interestedin some "extras" modules which are collected in the [Swift Metrics Extras](https://github.com/apple/swift-metrics-extras) repository.
```

?

Author: Thanks :)

> You may also be interestedin some "extras" modules which are collected in the [swift metrics extras](https://github.com/apple/swift-metrics-extras) repository.

Reviewer 2: ```suggestion
You may also be interested in some "extras" modules which are collected in the [swift metrics extras](https://github.com/apple/swift-metrics-extras) repository.
```

> Currently the repository includes: `SystemMetrics` allowing reporting of a node's CPU and memory metrics etc.

Reviewer: ```suggestion
Currently the repository includes `SystemMetrics` which allows reporting of a node's CPU and memory metrics etc.
```

Reviewer 2: not sure its worth detailing what extra includes as it won't scale to update both places. maybe just mention the kind of things that extra may include.

Author: Yeah that's a concern -- I'll just link to it then

> Currently the repository includes: `SystemMetrics` allowing reporting of a node's CPU and memory metrics etc.

Author: ```suggestion
```
